Modules/System/Barcode/src/Barcode Font Provider/BarcodeFontProvider.Enum.al

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary> 
+/// The available barcode font providers.
+/// </summary>
+enum 9203 "Barcode Font Provider" implements "Barcode Font Provider"
+{
+    Access = Public;
```

Comment:
Version 17 does not allow for access modifiers on enums. We're going to keep this for the 18 release, but for this check-in, please remove to make compilation succeed.